client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -0,0 +1,168 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'gridicons';
+import FoldableCard from 'components/foldable-card';
+import EllipsisMenu from 'components/ellipsis-menu';
+import PopoverMenuItem from 'components/popover/menu-item';
+import PopoverMenuSeparator from 'components/popover/menu-separator';
+import Gravatar from 'components/gravatar';
+
+class ActivityLogItem extends Component {
+
+	static propTypes = {
+		title: PropTypes.string,
+		subTitle: PropTypes.string,
+		className: PropTypes.string,
+		icon: PropTypes.string,
+		status: PropTypes.string,
+		user: PropTypes.object,
+		onClick: PropTypes.func,
```

Comment:
Yep!  For reference, see how it's used in the giant PR https://github.com/Automattic/wp-calypso/pull/10862/files#diff-0f2721fb654a62667b16edda79374e75R120